### PR TITLE
Disp work gatherer docs

### PR DESCRIPTION
**Pull request by arbulu89**
Docs for the `disp+work` gatherer:
https://github.com/trento-project/agent/pull/281

### CODE REVIEW

File: guides/gatherers.md [hunk 1]
```diff
@@ -24,13 +24,14 @@ Here's a collection of built-in gatherers, with information about how to use the
...
 | [`package_version`](#package_version)   | [trento-project/agent/../gatherers/packageversion.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/packageversion.go)   |
 | [`passwd`](#passwd)                     | [trento-project/agent/../gatherers/passwd.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/passwd.go)                   |
 | [`sapcontrol`](#sapcontrol)             | [trento-project/agent/../gatherers/sapcontrol.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapcontrol.go)           |
-| [`saphostctrl`](#saphostctrl)           | [trento-project/agent/../gatherers/saphostctrl.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/saphostctrl.go)         |
+| [`saphostctrl`](#saphostctrl)           | [trento-project/agent/../gatherers/saphostctrl.go](https://github.com/trento-project/agent/blob/main/inAternal/factsengine/gatherers/saphostctrl.go)         |
```
Comment (EMaksy) [inline guides/gatherers.md:34]: Typo in the link
`inAternal`should be `internal` right?

**Review by EMaksy — CHANGES_REQUESTED** [review]
Hey man besides the typo/broken link nothing to add, well done :D

**Review by EMaksy — APPROVED** [review]
LGTM :D

**Review by rtorrero — APPROVED** [review]
LGTM

**Pull request merged**